SPECS/python-psycopg2/python-psycopg2.spec

```diff
@@ -95,7 +104,14 @@ rm -r /home/postgres/data &>/dev/null ||:
 %defattr(-,root,root,-)
 %{python3_sitelib}/*
 
+%files doc
+%license LICENSE
+%doc doc
+
 %changelog
+* Tue Jul 13 2021 Muhammad Falak Wani <mwani@microsoft.com> - 2.7.5-6
+- Enable subpackage python-psycopg2-doc
```

Comment:
Please also mention where we got the new spec parts from - same comment as in #1126.